build.gradle

```
@@ -286,6 +286,7 @@ configure(rootProject) {
 			project.sourceSets.main.compileClasspath
 		})
 
+		exclude '**/sample/**'
```

I wonder if we shouldn't narrow it down to the sample app package? Would this work?
```suggestion
		exclude 'org.springframework.cloud.appbroker.sample**'
```

---

That would work. I was thinking this configuration would be applicable for other project with sample app as well. Is it common to have production code reside in a `sample` package?

---

We shouldn't have to worry about other production sample apps, since those shouldn't be published dependencies (artifacts).

---

@royclarkson applied your comment in pull request #136